app/src/apdu_handler.c

```diff
@@ -77,6 +135,49 @@ __Z_INLINE void handle_getversion(__Z_UNUSED volatile uint32_t *flags, volatile
     THROW(APDU_CODE_OK);
 }
 
+__Z_INLINE void handleGetAddr(volatile uint32_t *flags, volatile uint32_t *tx, uint32_t rx) {
+    zemu_log("handleGetAddr\n");
+    uint8_t len = extractHRP(rx, OFFSET_DATA);
```

Comment:
Do we need a configurable HRP here? If it's not needed, we can simplify this a little bit